CHANGES

```diff
@@ -1,4 +1,5 @@
 [1.10.1]
+- [BREAKING CHANGE]: Removed TiledMapTile default properties (terrain, probability, type) from the properties array and added them as getter/setters to the tile classes.
```

Comment:
maybe add a line (API addition) to say that builtin 'type' field is now supported?